config/routes.rb

```diff
@@ -1,2 +1,4 @@
 Rails.application.routes.draw do
+  get ":slug/next", to: "redirections#next"
```

Comment:
> Also, this allows people to have multiple subdomains that hook into the webring, i.e. example.com and blog.example.com could both use the example slug, instead of having multiple sites listed for each redirection or multiple sites from the same domain in the ring.

Duh, I forgot about that. Cool.